csharp-basics/exercises/MiniProjects/Minesweeper/Minesweeper/Core/Cell.cs

```diff
@@ -39,21 +36,73 @@ public void SetupDesign()
 
         public void OnFlag()
         {
+            if (CellType == CellType.Mine)
+            {
+                CellType = CellType.FlaggedMine;
+            }
+            else
+            {
+                CellType = CellType.Flagged;
+            }
+
+            Text = "?";
+        }
+
+        public void OffFlag()
```

Comment:
Kas ir domāts ar OffFlag? Ko šī metode dara? Pēc nosaukuma nevar saprast

Reply:
OnFlag() uzliek karogu. (Šis bija sagatavē).
OffFlag() noņem karogu.